ms-inventory/routes/common.go

```diff
@@ -78,47 +82,54 @@ func GetAuditLogEntryByID(auditEntryID string) (auditLogEntry AuditLogEntry, aud
 }
 
 // DeleteInventory will reset the content of the inventory JSON file
-func DeleteInventory() error {
-	return WriteJSON(InventoryFileName, Products{Data: []Product{}})
+func (c *Controller) DeleteInventory() error {
+	c.lc.Info("Inventory JSON content reset")
+	return c.WriteJSON(InventoryFileName, Products{Data: []Product{}})
 }
 
 // DeleteAuditLog will reset the content of the audit log JSON file
-func DeleteAuditLog() error {
-	return WriteJSON(AuditLogFileName, AuditLog{Data: []AuditLogEntry{}})
+func (c *Controller) DeleteAuditLog() error {
+	c.lc.Info("Audit Log JSON content reset")
+	return c.WriteJSON(AuditLogFileName, AuditLog{Data: []AuditLogEntry{}})
 }
 
 // WriteJSON is a shorthand for writing an interface to JSON
-func WriteJSON(fileName string, content interface{}) error {
+func (c *Controller) WriteJSON(fileName string, content interface{}) error {
+	c.lc.Debugf("Wrote: %s to Inventory JSON: %s", content, fileName)
 	return utilities.WriteToJSONFile(fileName, content, 0644)
 }
 
 // WriteInventory is a shorthand for writing the inventory quickly
-func (inventoryItems *Products) WriteInventory() error {
-	return WriteJSON(InventoryFileName, inventoryItems)
+func (c *Controller) WriteInventory() error {
+	c.lc.Debugf("Wrote: %s to Inventory: %s", c.inventoryItems, InventoryFileName)
+	return c.WriteJSON(InventoryFileName, c.inventoryItems)
 }
 
 // WriteAuditLog is a shorthand for writing the audit log quickly
-func (auditLog *AuditLog) WriteAuditLog() error {
-	return WriteJSON(AuditLogFileName, auditLog)
+func (c *Controller) WriteAuditLog() error {
+	c.lc.Debugf("Wrote: %s to Inventory: %s", c.auditLog, AuditLogFileName)
+	return c.WriteJSON(AuditLogFileName, c.auditLog)
 }
 
 // DeleteInventoryItem deletes an inventory item matching the
 // specified SKU
-func (inventoryItems *Products) DeleteInventoryItem(inventoryItem Product) {
-	for i, item := range inventoryItems.Data {
+func (c *Controller) DeleteInventoryItem(inventoryItem Product) {
+	for i, item := range c.inventoryItems.Data {
 		if item.SKU == inventoryItem.SKU {
-			inventoryItems.Data = append(inventoryItems.Data[:i], inventoryItems.Data[i+1:]...)
+			c.inventoryItems.Data = append(c.inventoryItems.Data[:i], c.inventoryItems.Data[i+1:]...)
+			c.lc.Debugf("Deleted: %s from inventory", inventoryItem.SKU)
 			break
 		}
 	}
 }
 
 // DeleteAuditLogEntry deletes an audit log entry item matching the
 // specified EntryID
-func (auditLog *AuditLog) DeleteAuditLogEntry(auditLogEntry AuditLogEntry) {
-	for i, item := range auditLog.Data {
+func (c *Controller) DeleteAuditLogEntry(auditLogEntry AuditLogEntry) {
+	for i, item := range c.auditLog.Data {
 		if item.AuditEntryID == auditLogEntry.AuditEntryID {
-			auditLog.Data = append(auditLog.Data[:i], auditLog.Data[i+1:]...)
+			c.auditLog.Data = append(c.auditLog.Data[:i], c.auditLog.Data[i+1:]...)
+			c.lc.Debugf("Deleted: %s from audit log", auditLogEntry.AuditEntryID)
```

Comment:
Why Debug when other deletes are Info? 
Normally these should al be Debug to avoid spamming the logs unless the code runs very infrequent.

Reply:
changed to debug